internal/appsec/emitter/ossec/lfi.go

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+package ossec
+
+import (
+	"io/fs"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo"
+)
+
+type (
+	// OpenOperation type embodies any kind of function calls that will result in a call to an open(2) syscall
+	OpenOperation struct {
```

Comment:
There should be a `Finish` method on this type, in order to match the API design of other operation structs.

Reply:
I get it, but that makes the one the only operation type to NOT have the `Finish` method...

Reply:
Discussed offline -- there is another precedent at least; and this code will be subjected to a significant refactor soon; so let's drop this.